TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utilities/Slides.java

```diff
@@ -9,117 +9,166 @@
 import com.qualcomm.robotcore.hardware.HardwareMap;
 
 public class Slides {
-    DcMotor slideMotor;
+
+    // to go up
+    //Looking from front, Left (acc on the right in this view), must go clockwise
+    // Right (Acc on the left in this view), must go counter clockwise
+    // Positive power is counter clockwise,
+
     //position at initial
-    int pos;
+    int Lpos;
+    int Rpos;
+    //Current state of slide. 0 - idle, 1 - up, 2 - down
     int state;
 
-    //unknown, values go into negative, 0 > lowheight > midheight > maxheight
-    int maxheight = -8000;
-    int midheight = -6847;
-    int lowheight = -3800;
+    //Preset heights,
+    // TO DO: CALIBRATE
+    int maxheight = 8000;
```

Comment:
Let's have relevant constants but another branch should be responsible for this.